cli/cmd/version.go

```diff
@@ -17,6 +19,6 @@ var versionCmd = &cobra.Command{
 example: CFT CLI version v1.0.0`,
 	Args: cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
-		cmd.Println("CFT CLI version", Version)
+		fmt.Println(Version)
```

Comment:
This change is not passing the [test case here](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/cli/cmd/version_test.go#L16)

Reply:
Oops! Fixing: #701